go/consensus/tendermint/db/badger/badger.go

```diff
@@ -474,11 +474,24 @@ func (ba *badgerDBBatch) WriteSync() error {
 
 	for _, cmd := range ba.cmds {
 		if err := cmd.Execute(tx); err != nil {
-			ba.db.logger.Error("failed to execute command in Tx",
-				"err", err,
-				"cmd", cmd,
-			)
-			return err
+			if err == badger.ErrTxnTooBig {
```

Comment:
This should get a backport to 22.x probably.

Reply:
@abukosek Can you extract the fix into a separate PR and also do a backport?

Reply:
PR with the fix: https://github.com/oasisprotocol/oasis-core/pull/4648
Backport will be done after the PR is merged.